ls1intum/Artemis
# `Development`: Short Answer Number Input

### Checklist
#### General


#### Server
#### Client

### Motivation and Context



[#5459](https://github.com/ls1intum/Artemis/issues/5459)

### Description


Added Spot Type Number to Short Answer Quiz Exercise

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student
- 1 Course

1. Log in to Artemis as Instructor
2. Navigate to Course Management
3. Select the Course
4. Navigate to Exercises
5. Click **+ Create Quiz**
6. Input Quiz Title
7. Click **+ Add Short-Answer Question**
8. Input Question Title
9. Clear the question template
10. Click **Add Number Spot**
11. Input the option either in range format (e.g 0.5 - 10) or a single number (e.g 8.0)
12. Click Save
13. Log out and Log in as Student
14. Click **Start Quiz**
15. Input to the spot should be limited to number only
16. Submit

### Review Progress






#### Code Review
#### Manual Tests
#### Exam Mode Test

### Test Coverage


| Class/File | Branch | Line |
|------------|-------:|-----:|


### Screenshots
